cmrtlib/linux/CMakeLists.txt

```diff
@@ -90,8 +77,25 @@ endif()
 set_target_properties( igfxcmrt PROPERTIES 
   FOLDER CM_RT 
   COMPILE_DEFINITIONS "${CMRT_DEFINES}" 
-  OUTPUT_NAME "igfxcmrt${TARGET_MODIFIER}"
+  OUTPUT_NAME "igfxcmrt"
   )
-set_target_properties( igfxcmrt PROPERTIES PREFIX "")
+
+# set the version, aligned with media driver
+if(NOT DEFINED MEDIA_VERSION OR MEDIA_VERSION STREQUAL "")
+    set(MEDIA_VERSION "1.0.0")
+endif()
+set(CMRT_VERSION ${MEDIA_VERSION})
+string(REPLACE "." ";" CMRT_VERSION_LIST ${CMRT_VERSION})
+list(GET CMRT_VERSION_LIST 0 CMRT_VERSION_MAJOR)
```

Comment:
Well, this will not do. You can't actually delegate setting soversion on Linux outside of the build scripts. The critical point here is major version: thru it you communicate backward _incompatible_ changes. This means that you can't let it go wrong. Most distributions will not be aware that they need to setup it at all. If someone will change it - we will never guess what they will chose. Thus, the correct approach would be to simply hardcode versioning here like:
```
set(CMRT_VERSION_MAJOR 1)
set(CMRT_VERSION_MINOR 0)
set(CMRT_VERSION_PATCH 0)
set(CMRT_VERSION ${CMRT_VERSION_MAJOR}.${CMRT_VERSION_MINOR}.${CMRT_VERSION_PATCH}.)
```
And make sure that you
* Bump MAJOR version up on each backward incompatible uAPI change
* Bump MINOR version up on each backward compatible uAPI change
* Bump up PATCH version on each release bug fix.